portafly/package.json

```diff
@@ -24,13 +24,26 @@
     "@types/react-dom": "^16.9.0",
     "@types/react-router": "^5.1.3",
     "@types/react-router-dom": "^5.1.3",
-    "date-fns": "^2.9.0"
+    "@typescript-eslint/eslint-plugin": "^2.23.0",
+    "@typescript-eslint/parser": "^2.22.0",
+    "date-fns": "^2.9.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-typescript": "^7.0.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-jest": "^23.8.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.19.0",
+    "eslint-plugin-react-hooks": "^2.5.0",
+    "stylelint": "^13.2.0",
+    "stylelint-config-sass-guidelines": "^7.0.0",
+    "stylelint-config-standard": "^20.0.0"
```

Comment:
So most of these dependencies are not needed for StandardJS?

Reply:
Stylelint is not Eslint-related (I simply haven't added to the other PR yet) and the rest of dependencies come already with Standard. Some like `jest` and `hooks` are not used by standard.